assets/js/application_forms.js

```diff
@@ -183,12 +183,27 @@ $(document).ready(function(){
 $(document).on('change blur keydown paste input', '.number', function () {
     var sum = 0;
     var totalCost = 0;
+    var isPremium = false
     $('body').find('.number').each(function () {
         sum += Number($(this).val());
         totalCost = sum * documentCost;
+
+        // Check if the 'max' attribute is defined for the doc quantity input
+        // if so it's a premium application
+        var maxAttr = $(this).attr('max');
+        if (typeof maxAttr !== 'undefined' && maxAttr !== false) {
+          isPremium = true;
```

Comment:
This may always be true since max is defined on both non/premium document.ejs pages

Reply:
I've removed the max attribute from applicationForms/documentQuantity.ejs, since its only used when the flag to limit the standard applications is turned on. This is old functionality which we can remove in a cleanup exercise.